apps/cli/mcp/server.go

```diff
@@ -34,6 +34,7 @@ func (s *DaytonaMCPServer) Start() error {
 func (s *DaytonaMCPServer) addTools() {
 	createSandboxTool := mcp.NewTool("create_sandbox",
 		mcp.WithDescription("Create a new sandbox with Daytona"),
+		mcp.WithString("id", mcp.Description("If sandbox ID is provided it is first checked if it exists and is running, if so the existing sandbox will be used. However, model is not able to provide custom sandbox ID but only the ones Daytona commands return and should always leave ID field empty if a new sandbox wants to be created.")),
```

Comment:
`If a sandbox ID is provided it is first checked if it exists and is running, if so, the existing sandbox will be used. However, a model is not able to provide custom sandbox ID but only the ones Daytona commands return and should always leave ID field empty if the intention is to create a new sandbox.`